### PR TITLE
Updated the doco to package default or custom binaries for a lambda function

### PR DESCRIPTION
**Pull request by vikas027**
Updated the doco to package default or custom binaries for a lambda function
This is just for advanced users, but still, it is good to have it for clarity

# Issue Type
- Documentation Pull Request

### CODE REVIEW

File: CUSTOM_BUILDS.md [hunk 1]
```diff
@@ -83,6 +83,8 @@ your CloudFormation or Terraform stack to use those new binaries.
 
 ## Using your own binaries in AWS Lambda ##
 
+You can either follow the below process to upload the lambda package to a S3 bucket, or compile the binaries as shown [above](#compiling-the-binaries-locally) and create a lambda deployment package by zipping (`zip autospotting.zip autospotting`) the built binary.
```
[the remark this answers is not in the document]
Comment (vikas027) [reply CUSTOM_BUILDS.md]: I have fixed this now.

File: CUSTOM_BUILDS.md [hunk 2]
```diff
@@ -83,6 +83,11 @@ your CloudFormation or Terraform stack to use those new binaries.
 
 ## Using your own binaries in AWS Lambda ##
 
+You can either follow the below process to upload the lambda package to a
+S3 bucket, or compile the binaries as shown [above](#compiling-the-binaries-locally)
+and create a lambda deployment package by zipping (`zip autospotting.zip autospotting`)
```
Comment (cristim) [inline CUSTOM_BUILDS.md]: We have the `make archive` target that already creates the zip file, we can just use that.

The file can then be found under the `build` directory.

The `BUCKET_NAME=my-bucket make upload` target will build and then automatically upload it to your own S3 bucket.

Comment (vikas027) [reply CUSTOM_BUILDS.md]: Ah cool, I didn't know that. Let me add that to the doco.

**Comment by vikas027** [comment]
Sure, that makes sense but what if someone does not wants to use an S3 bucket. I think it does not harm to keep that optional step, just in case someone wants some flexibility.

**Pull request merged**